en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
 
-There are a variety of resources that can help researchers think through these and other ethical issues. The University of California at Berkeley hosted a conference on ethical and legal topics in June 2020: Building LLTDM - Legal Literacies for Text Data Mining. Review [the LLTDM website](https://buildinglltdm.org/), as well as the [Association of Internet Researcher’s Ethics page](https://aoir.org/ethics/) and Annette Markham's [Impact Model for Ethics: Notes from a Talk](https:// annettemarkham.com/2017/07/impact-model-ethics/).
+Does researching a group of users with whom the researcher is less culturally familiar open the door to causing unintentional harm? Who speaks for communities being researched online? Researchers should consider their own cultural context and limitations in interpreting discourse from other cultures and contexts.
 
-## Video Selection
-The most direct way to make your selection of videos for research is to visit the YouTube site, and capture a list of video IDs their URL. A video’s ID is the set of alphanumeric characters that appear in the URL immediately after `watch?v=`. For example, in the illustration below, the video ID is circled in red: `24xsqyMcpRg`. Video IDs are constant and do not change over time. 
+There are no clear answers to these challenging questions, but they should nonetheless be asked and studied. Digital Humanities researchers should fully incorporate ethical thinking into the use of their code: not everything that could be mined, analyzed, published, and visualized from YouTube should be.
 
-{% include figure.html filename="en-or-text-mining-youtube-comments-02.png" alt="Screenshot of YouTube video with video ID in browser link circled in red" caption="Figure 2: Screenshot of YouTube video with video ID in browser link circled in red" %}
+There are a variety of resources that can help researchers think through such ethical issues. The University of California at Berkeley hosted a conference on ethical and legal topics in June 2020, recorded in the open access book [_Building Legal Literacies for Text Data Mining_](https://berkeley.pressbooks.pub/buildinglltdm/). Review also [the LLTDM website](https://buildinglltdm.org/), as well as the [Association of Internet Researcher’s Ethics page](https://aoir.org/ethics/) and Annette Markham's [Impact Model for Ethics: Notes from a Talk](https://annettemarkham.com/2017/07/impact-model-ethics/).

```

Comment:
I saw that an open access book summarising the contents of that conference was later published, and a link to it – let me know if this works for you.

Reply:
That's great, thanks!